src/imageio/format/avif.c

```diff
@@ -402,14 +458,16 @@ int write_image(struct dt_imageio_module_data_t *data,
         break;
     }
 
+    if (nclx.colourPrimaries != AVIF_NCLX_COLOUR_PRIMARIES_UNKNOWN) {
```

Comment:
Would be nice to fix the style on this file. The { and } in dt are always on their own line. I haven't noticed that before.
So we should have:

```
if()
{
}
else
{
}
```

Likewise for switch

```
switch()
{
}
```
I will merge this for 3.2 as it seems safe. The reformatting can come later.